### PR TITLE
Workflows - main screen - side navigation #96

### PR DESCRIPTION
**Pull request by jozefbakus**
implements: #96

### CODE REVIEW

**Comment by kevinwallimann** [comment]
I would name the branches like this: feature/{issue number}-{some description}, so e.g. feature/96-workflows-navigation.

**Comment by kevinwallimann** [comment]
I think the sidenav should be resizable. For really long workflow names, there will be multiple word wraps, while there would be enough space on the right side, especially on a big screen

File: ui/src/app/components/workflows/workflows.component.ts [hunk 1]
```diff
@@ -13,18 +13,41 @@
...
+  projects: ProjectModel[] = [];
+  workflows: WorkflowModel[] = [];
 
-  ngOnInit(): void {
+  constructor(private store: Store<AppState>) {
+    this.store.dispatch(new InitializeWorkflows());
+  }
+
+  ngAfterViewInit(): void {
+    this.workflowsSubscription = this.store.select(selectWorkflowState).pipe(skip(1)).subscribe((state) => {
```
Comment (kevinwallimann) [inline ui/src/app/components/workflows/workflows.component.ts]: What is the `skip(1)` for?

File: ui/src/app/stores/workflows/workflows.effects.spec.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {TestBed} from '@angular/core/testing';
+// import {RunsEffects} from "./work.effects";
```
Comment (kevinwallimann) [inline ui/src/app/stores/workflows/workflows.effects.spec.ts]: Some unused imports

**Review by kevinwallimann — COMMENTED** [review]
Please see my comments. Code is really good, I've got almost nothing to say

File: ui/src/app/components/workflows/workflows.component.ts (hunk 1 above)
Comment (kevinwallimann) [reply ui/src/app/components/workflows/workflows.component.ts]: Nevermind. (Removed in #103)

File: ui/src/app/stores/workflows/workflows.effects.spec.ts (hunk 2 above)
Comment (jozefbakus) [reply ui/src/app/stores/workflows/workflows.effects.spec.ts]: Done

**Comment by jozefbakus** [comment]
> I would name the branches like this: feature/{issue number}-{some description}, so e.g. feature/96-workflows-navigation.

With new branches, I will follow this.

**Comment by jozefbakus** [comment]
> I think the sidenav should be resizable. For really long workflow names, there will be multiple word wraps, while there would be enough space on the right side, especially on a big screen

This was partially fixed here https://github.com/AbsaOSS/hyperdrive-trigger/pull/103
but still not perfect. I've created an issue for this problem https://github.com/AbsaOSS/hyperdrive-trigger/issues/105

**Review by kevinwallimann — APPROVED** [review]

**Pull request merged**